src/utils/releaseNotes.js

```diff
@@ -30,4 +35,45 @@ function getPrNumbersFromReleaseNotes(releaseNotes) {
   return [...new Set(allPrNumbers)]
 }
 
+exports.getPRBody = (
```

Comment:
for consistency, let's declare a plain function and then export it at the bottom of this module, as we're doing with the other function